### PR TITLE
docs: documentation about balsamic filters

### PR DESCRIPTION
**Pull request by ashwini06**
### This PR:

Added: Documentation about the BALSAMIC variant calling filters. 
Part of https://github.com/Clinical-Genomics/project-planning/issues/158

Changed:
Updated the BALSAMIC[ readthedocs workflow resources](https://balsamic.readthedocs.io/en/latest/bioinfo_softwares.html)

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Review by ivadym — APPROVED** [review]
Great job Ashwini @ashwini06 🥇

File: docs/balsamic_filters.rst [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+..  figure:: images/vcf_filters.png
+    :width: 500px
+
+    Pre-call filters applied by the `Vardict` variant-calleer is listed out in the VCF header
+
+
+In the VCF file, `FILTER` status is `PASS` if this position has passed all filters, i.e., a call is made at this position. Otherwise,
+if the site has not passed all filters, a semicolon-separated list of codes for filters that fail. e.g., `p8;pSTD` might
+indicate that at this site, the mean position in reads less than 8 and position in reads has a standard deviation of 0.
+In BALSAMIC, this VCF file is named as `*.all.vcf.gz` (eg: `SNV.somatic.$case_id.vardict.all.vcf.gz`)
```
Comment (khurrammaqbool) [inline docs/balsamic_filters.rst:22]: <CASE_ID>

**Comment by ashwini06** [comment]
@keyvanelhami @khurrammaqbool @henrikstranneheim @moahaegglund : Please review this document https://github.com/Clinical-Genomics/BALSAMIC/blob/doc/balsamic_filters/docs/balsamic_filters.rst. 

Written from technical point of view, feel free to comment on the content structure, if it is understandable from customer perspective. Idea is to hyperlink to the related subsections of this document to the delivery report (eg: for TGA somatic, direct link to [vardict filtering](https://github.com/Clinical-Genomics/BALSAMIC/blob/doc/balsamic_filters/docs/balsamic_filters.rst#vardict_filtering), for WGS somatic, link to[ Sentieon TNscope filtering](https://github.com/Clinical-Genomics/BALSAMIC/blob/doc/balsamic_filters/docs/balsamic_filters.rst#sentieons-tnscope))

**Review by khurrammaqbool — APPROVED** [review]
Some minor comments.

File: docs/balsamic_filters.rst [hunk 2]
```diff
@@ -0,0 +1,175 @@
+***********************************
+BALSAMIC Variant Calling Algorithms
+***********************************
+
+In BALSAMIC, various bioinfo tools are integrated for reporting somatic and germline variants. Also, the choice of these tools differs between the type of analysis
+for eg: `Target Genome Analysis (TGA)` or `Whole Genome Sequencing (WGS)`. Various filters are applied at different levels to report high-confidence variant calls.
+`Pre-call filtering` is where the variant-calling tool decides not to emit a variant line to the VCF file.
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_filters.rst]: This line is modified in the recent version, but edited the word `emit` to `call` for more clarification

File: docs/balsamic_filters.rst [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+`Post-call filtering` is where a variant is emitted along with ancillary metrics, such as quality and depth, which are then used for further filtering.
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_filters.rst]: Fixed!

File: docs/balsamic_filters.rst [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+**Whole Genome Sequencing (WGS)**
+**********************************
+
+**Sentieon's TNscope**
+======================
+
+BALSAMIC utilizes `TNscope` algorithm for the variant calling of somatic SNV/INDELS in WGS samples.
+The `TNscope <https://www.biorxiv.org/content/10.1101/250647v1.abstract>`_ algorithm performs the somatic variant calling on the tumor-normal or the tumor-only samples, using a Haplotyper algorithm.
+The mathematical model of `TNscope` is based on `Mutect` with improved accuracy and performance.
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_filters.rst]: Removed!

File: docs/balsamic_filters.rst [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+    Maximum AF(tumor) < 1
+
+*GNOMADAF_POPMAX*: Maximum Allele Frequency across populations
+
+::
+
+    GNOMADAF_popmax <= 0.001 (or) GNOMADAF_popmax == "."
+
+
+*Normalized base quality scores*:  The sum of base quality scores for each allele (QSS) is divided by the allelic depth of alt and ref alleles (AD)
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_filters.rst:152]: this is wrong, in the recent version, you will find the right one.

**Pull request merged**